Integrations/ZeroFox/ZeroFox.py

```diff
@@ -0,0 +1,510 @@
+        f'Alert: {alert_id} has been opened successfully.',
+        context,
+        raw_response={}
+    )
+
+
+def alert_request_takedown(alert_id):
+    url_suffix: str = f'/alerts/{alert_id}/request_takedown/'
+    response_content = http_request('POST', url_suffix)
+    return response_content
+
+
+def alert_request_takedown_command():
+    alert_id: int = demisto.args().get('alert_id')
+    alert_request_takedown(alert_id)
+    context = get_alert_context_no_war_room(alert_id)
+    return_outputs(
+        f'Alert: {alert_id} has been requested to be taken down successfully.',
+        context,
+        raw_response={}
+    )
+
+
+def alert_cancel_takedown(alert_id):
+    url_suffix: str = f'/alerts/{alert_id}/cancel_takedown/'
+    response_content = http_request('POST', url_suffix)
+    return response_content
+
+
+def alert_cancel_takedown_command():
+    alert_id: int = demisto.args().get('alert_id')
+    alert_cancel_takedown(alert_id)
+    context = get_alert_context_no_war_room(alert_id)
+    return_outputs(
+        f'Alert: {alert_id} has canceled takedown successfully.',
+        context,
+        raw_response={}
+    )
+
+
+def alert_user_assignment(alert_id, subject_email, subject_name):
+    url_suffix: str = f'/alerts/{alert_id}/assign/'
+    request_body: Dict = {
+        'subject_email': subject_email,
+        'subject': subject_name
+    }
+    response_content = http_request('POST', url_suffix, data=json.dumps(request_body))
+    return response_content
+
+
+def alert_user_assignment_command():
+    alert_id: int = demisto.args().get('alert_id')
+    subject_name: str = demisto.args().get('subject_name')
+    subject_email: str = demisto.args().get('subject_email')
+    alert_user_assignment(alert_id, subject_email, subject_name)
+    context = get_alert_context_no_war_room(alert_id)
+    return_outputs(
+        f'User: {subject_email} has been assigned to Alert: {alert_id} successfully.',
+        context,
+        raw_response={}
+    )
+
+
+def modify_alert_tags(alert_id, action, tags_list_string):
+    url_suffix: str = '/alerttagchangeset/'
+    tags_list_name: str = 'added' if action else 'removed'
+    tags_list: list = tags_list_string.split(',')
+    request_body: Dict = {
+        'changes': [
+            {
+                f'{tags_list_name}': tags_list,
+                'alert': alert_id
+            }
+        ]
+    }
+    response_content = http_request('POST', url_suffix, data=json.dumps(request_body))
+    return response_content
+
+
+def modify_alert_tags_command():
+    alert_id = demisto.args().get('alert_id')
+    action_string = demisto.args().get('action')
+    action = True if action_string == 'add' else False
+    tags_list_string = demisto.args().get('tags')
+    modify_alert_tags(alert_id, action, tags_list_string)
+    context = get_alert_context_no_war_room(alert_id)
+    return_outputs(
+        'Changes were successfully made.',
+        context,
+        raw_response={}
```

Comment:
return context here

Reply:
returned contents, I think it's the right thing to return here